package.json

```diff
@@ -52,10 +55,15 @@
     "@types/node": "^12.12.47",
     "@types/node-sass": "^4.11.1",
     "all-contributors-cli": "^6.16.0",
+    "axiosist": "^0.9.0",
     "copyfiles": "^2.3.0",
+    "cypress": "^4.9.0",
     "eslint": "^7.2.0",
+    "eslint-plugin-cypress": "^2.11.1",
     "eslint-plugin-vue": "^7.0.0-alpha.6",
+    "jest": "^26.1.0",
     "jsdom": "^16.2.2",
+    "mock-json-schema": "^1.0.8",
```

Comment:
I think that your schemas need fixing for this as allOf gets to this path: https://github.com/anttiviljami/mock-json-schema/blob/master/src/mock.ts#L7

Reply:
Thanks for the link, I'll fix the allOf to anyOf.